src/iexamine.h

```diff
@@ -14,6 +14,7 @@ class game;
 class item;
 class player;
 class map;
+class tripoint;
```

Comment:
> src/iexamine.h:17:1: note: did you mean struct here?

Yes, my g++ knows what I meant. I should probably compile with clang more often. Thanks fixing it.